### PR TITLE
[4.0] Add back button to the transitions and stages view button toolbars

### PR DESCRIPTION
**Pull request by bembelimen**
### Summary of Changes
A few weeks ago the submenu was removed from the workflow and it is very  circuitous to switch from stages to transitions. This improves the workflow with some "back" buttons.

### Testing Instructions
Go to the stages/transitions view and check the back button on the top left toolbar

### CODE REVIEW

**Comment by wojsmol** [comment]
@zero-24 @SniperSister Please check this tests failure as this is related to rips.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on ae32cfa44e9de9878133ab94673c847e8dcb1911<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23460">issues.joomla.org/tracker/joomla-cms/23460</a>.</sub>

**Comment by brianteeman** [comment]
Using arrow-left makes no sense if the site is RTL

**Comment by bembelimen** [comment]
You're right, what icon would you use?

**Comment by brianteeman** [comment]
You can still use an arrow but make the direction conditional on the language - see j-toggle-sidebar for an example https://github.com/joomla/joomla-cms/blob/026d83f4296190286dd71c03a8d796bd818da3b9/administrator/templates/isis/js/template.js#L273

**Comment by zero-24** [comment]
Rips is green now.

**Comment by Hackwar** [comment]
I have tested this item :white_check_mark: successfully on 1b508c410969cf05c3a5956546c2dde87eee49e4<br /><br />I've tested the functionality. I can't say anything regarding the icon.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23460">issues.joomla.org/tracker/joomla-cms/23460</a>.</sub>

**Comment by infograf768** [comment]
@bembelimen 
You can use:
```
		$arrow = Factory::getLanguage()->isRtl() ? 'arrow-right' : 'arrow-left';

		ToolbarHelper::link('index.php?option=com_workflow&view=workflows&extension=' . $this->escape($this->workflow->extension),
			'JTOOLBAR_BACK', $arrow);
```

**Comment by zero-24** [comment]
PHPCS issues :) http://ci.joomla.org/joomla/joomla-cms/13521/8

File: administrator/components/com_workflow/View/Stages/HtmlView.php [hunk 1]
```diff
@@ -160,6 +160,10 @@ protected function addToolbar()
 		ToolbarHelper::title(Text::sprintf('COM_WORKFLOW_STAGES_LIST', $this->escape($workflow)), 'address contact');
 
 		$isCore = $this->workflow->core;
+		$arrow = Factory::getLanguage()->isRtl() ? 'arrow-right' : 'arrow-left';
```
Comment (infograf768) [inline administrator/components/com_workflow/View/Stages/HtmlView.php]: align = to the one above

**Comment by bembelimen** [comment]
Sorry was to slow, thanks everyone for the improvements!

**Pull request merged**